e2e-tests/cypress/integration/tests/a11y.e2e_spec.js

```diff
@@ -0,0 +1,274 @@
+import { auth } from '../pageobjects/authentication.pageobject.js'
+import { menu, verify } from '../pageobjects/common.pageobject.js'
+import { overview } from '../pageobjects/overview.pageobject.js'
+import { myprofile } from '../pageobjects/myprofile.pageobject.js'
+import { history } from '../pageobjects/history.pageobject.js'
+import { buy, travelTime } from '../pageobjects/buyticket.pageobject.js'
+
+/**
+ * Accessibility checks using axe (https://github.com/dequelabs/axe-core)
+ */
+
+describe("a11y check authentication", () => {
```

Comment:
Det ligger inne en prettier config i root, sikkert hensiktsmessig å bruke samme formatering her. Jeg kan sette opp noe commit hooks for å sikre det for så vidt

Reply:
Kjørt prettier på fila